@navikt/core/react/src/datepicker/datepicker.stories.tsx

```diff
@@ -0,0 +1,257 @@
+            )}
+          </>
+        )}
+      </Comp>
+    </div>
+  );
+};
+
+Default.args = {
+  yearSelector: false,
+  disableWeekends: false,
+  showWeekNumber: false,
+  inputfield: true,
+  standalone: false,
+  openOnFocus: true,
+};
+
+export const YearSelector = (props) => (
+  <div>
+    <DatePicker
+      yearSelector
+      fromDate={new Date("Aug 23 2018")}
+      toDate={new Date("Aug 23 2022")}
+    >
+      <DatePicker.Input label="Velg dato" size={props.size}></DatePicker.Input>
+    </DatePicker>
+  </div>
+);
+
+export const DisabledDays = (props) => (
+  <div>
+    <DatePicker disabled={disabledDays} disableWeekends locale="en">
+      <DatePicker.Input label="Velg dato" size={props.size}></DatePicker.Input>
+    </DatePicker>
+  </div>
+);
+
+export const ShowWeekNumber = (props) => (
+  <div>
+    <DatePicker showWeekNumber>
+      <DatePicker.Input label="Velg dato" size={props.size}></DatePicker.Input>
+    </DatePicker>
+  </div>
+);
+
+export const UseDatepicker = () => {
+  const { dayPickerProps, selectedDay, inputProps } = useDatepicker({
+    fromDate: new Date("Aug 23 2019"),
+    locale: "en",
+    openOnFocus: true,
+  });
+
+  useEffect(() => {
+    selectedDay && isValidDate(selectedDay) && console.log(selectedDay);
+  }, [selectedDay]);
+
+  return (
+    <div style={{ display: "flex", gap: "1rem" }}>
+      <DatePicker {...dayPickerProps}>
+        <DatePicker.Input {...inputProps} label="Velg dato" />
+      </DatePicker>
+    </div>
+  );
+};
+
+export const UseRangedDatepicker = () => {
+  const { dayPickerProps, fromInputProps, toInputProps, selectedRange } =
+    useRangeDatepicker({
+      fromDate: new Date("Aug 23 2019"),
+      openOnFocus: true,
+    });
+
+  useEffect(() => {
+    selectedRange && console.log(selectedRange);
+  }, [selectedRange]);
+
+  return (
+    <div style={{ display: "flex", gap: "1rem" }}>
+      <DatePicker {...dayPickerProps}>
+        <div style={{ display: "flex", gap: "1rem" }}>
+          <DatePicker.Input {...fromInputProps} label="Fra" />
+          <DatePicker.Input {...toInputProps} label="Til" />
+        </div>
+      </DatePicker>
+    </div>
+  );
+};
+
+export const NoPopover = () => (
+  <div>
+    <DatePicker.Standalone />
+  </div>
+);
+
+export const UserControlled = () => {
+  const [open, setOpen] = useState(false);
+  const id = useId();
+
+  return (
+    <div>
+      <DatePicker
+        mode="multiple"
+        open={open}
+        onClose={() => setOpen(false)}
+        id={id}
+      >
+        <Button
+          aria-controls={id}
+          aria-haspopup="grid"
+          onClick={() => setOpen((x) => !x)}
+        >
+          Legg til dager
+        </Button>
+      </DatePicker>
+    </div>
+  );
+};
+
+export const Validering = () => {
+  const { dayPickerProps, selectedDay, inputProps } = useDatepicker({
+    fromDate: new Date("Aug 23 2019"),
+  });
+
+  const [errorState, setErrorState] = useState<string | null>(null);
```

Comment:
`errorState` kan regnes ut fra `selectedDay`, så for å [minimere state handling](https://reactjs.org/docs/thinking-in-react.html#step-3-identify-the-minimal-but-complete-representation-of-ui-state) ville jeg unngått å mellomlagre den.

Reply:
Lurt, var en rask fix 🎸